uiowa/uiowa
# Hover color fix for black url button on yellow background banner

Fixes: #3767 

# How to test
1. Create banners with gold backgrounds and insert URL.
2. Hover to make sure a gold hover shows up on the black button.
3. Make sure other banners are okay.
